README.md

```diff
@@ -264,6 +266,99 @@ It is possible that a standby trying to activate is not able to immediately acqu
 it goes back into standby mode, but if it fails to receive heartbeats from an active server, it will try again to
 acquire the store lock. The interval is random but as of now set to a bit more than a second.
 
+## Static Channels aka Partitioning
+
+It is possible to limit the list of channels a server can handle. This can be used to:
+
+* Prevent creation of unwanted channels
+* Share the load between several servers running with the same cluster ID
+
+In order to do so, you need to enable the `static_channels` parameter in the configuration file (or `channels_partitioning` if
+you prefer), and also specify the list of allowed channels in the `channels` section of the `store_limits` configuration.
+
+Channels don't need to override any limit, but they need to be specified for the server to service only these channels.
+
+Here is an example:
+
+```
+static_channels: true
+store_limits: {
+    channels: {
+        "foo": {}
+        "bar": {}
+    }
+}
+```
+
+When defining static channels, the server will no longer check that it is the only server running on the network
+(with a given cluster ID). On the contrary, this configuration allows multiple servers with the same cluster ID
+to run on the network, but each server handles its own set of channels. ***Note however that in this mode,
+state is not replicated (as it will be in the full clustering feature to come). The only communication between
+servers is to try detect invalid channels configuration***.
+
+### A given channel must be defined in a single server
+
+When a server starts, it sends its list of channels to all other servers on the same cluster in an attempt
+to detect duplicate channels. When a server receives this list and finds that it has a static channel in
+common, it will return an error to the emitting server, which will then fail to start.
+
+However, on startup, it is possible that the undelying NATS cluster is not fully formed. The server would
+not get any response for the rest of the cluster and therefore start successully and service clients.
+Anytime a Streaming server detects that a NATS server was added to the NATS cluster, it will resend its list
+of channels. It means that currently running servers may suddendly fail with a message regarding duplicate channels.
+It also means that the same channel may have been created in different servers, the same subscription accepted
+in different servers (but only one would be able to process application acknowledgements), etc...
+
+***In order to avoid issues with channels existing on several servers, it is ultimately the responsability
+of the administrator to ensure that channels are unique.***
+
+### Wildcards
+
+NATS Streaming does not support sending or subscribing to wildcard channels (such as `foo.*`).
+
+However, it is possible to use wildcards to define the static channels that a server can handle.
+For instance, with the following configuration:
+```
+static_channels: true
+store_limits: {
+    channels: {
```

Comment:
I probably could have been a simple array, unfortunately that was not added in this PR and if changing now would break existing configuration files.